templates/explore/repo_list.tmpl

```diff
@@ -9,12 +9,17 @@
 					<span><i class="octicon octicon-repo-forked"></i></span>
 				{{else if .IsMirror}}
 					<span><i class="octicon octicon-repo-clone"></i></span>
+				{{else if eq .Owner.Visibility 3}}
```

Comment:
An extra test required because .Owner is not defined then an error occur ` <.Owner.Visibility>: can't evaluate field Visibility in type *models.User`

Possible solution:

```
				{{else if .Owner}}
					{{if eq .Owner.Visibility 3}}
			          <span class="text gold"><i class="octicon octicon-lock"></i></span>
					{{end}}
				{{end}}
````

Reply:
Thanks @scr34m I've just updated the template